AUTHORS.rst

```diff
@@ -0,0 +1,18 @@
+=======
+Credits
+=======
+
+Development Lead
+----------------
+
+* MIT Data To AI Lab <dailabmit@gmail.com>
```

Comment:
I would say the development lead should be an actual person, though I'm not sure if anyone is actually in this role?

Another possibility is to delete the `development lead` subsection entirely.